jep/200/README.adoc

```diff
@@ -99,19 +99,22 @@ in favor of the more general `ClassFilter.setDefault`, as is the system property
 
 The new filter has several levels of rules:
 
-* A `CustomClassFilter` implementation may override any behavior. This is discussed in the compatibility section.
+* A custom `ClassFilter` implementation may override any behavior. This is discussed in the compatibility section.
 * Classes mentioned in the original blacklist continue to be rejected.
 * Java arrays and enums are accepted, as these are special formats.
 * Java `Throwable` types are accepted, as they are commonly used in Remoting responses and it is impractical to enumerate every exception that might be thrown.
-* A class defined in Jenkins core, Remoting, or a Jenkins plugin is accepted. This does _not_ apply to classes defined in other JARs packaged in `WEB-INF/lib/*.jar`.
+* A class defined in Jenkins core, Remoting, or a Jenkins module/plugin is accepted. This does _not_ apply to classes defined in other JARs packaged in `WEB-INF/lib/*.jar`.
 * Some classes defined in test utilities, by mocking frameworks, etc., are accepted when running inside tests.
 * Classes defined in a `RemoteClassLoader` (so, already sent to this JVM from a presumably trusted source) are accepted.
   This is needed at least by CloudBees Jenkins Operations Center, and clients of the `master-to-master-api` plugin if there are any.
 * Any other class is accepted if it is explicitly mentioned in a list `whitelisted-classes.txt` bundled in Jenkins core. These consist of:
-** Basic Java utility types (roughly half).
+//oleg-nenashev: What does block us from allowing all types?
```

Comment:
It seems to me that the creation of comments like this hints at a process issue. There's no real destination for people to comment on accepted draft JEPs. CC @rtyler @bitwiseman

Reply:
*long protracted sigh*

that was the point of pull requests

Reply:
I could comment in a merged pull request if it helps

Reply:
General comments should go to the mailing list.

To answer your question, nothing. I added those types whose absence caused tests to fail. Nothing more, nothing less.